TestContainer/test/testReleaseNotes.js

```diff
@@ -4,18 +4,106 @@ const chaiHttp = require("chai-http");
 const { expect } = require("chai");
 
 chai.use(chaiHttp);
+
 var accessToken;
-const addressDelete = "/api/releasenote/1";
+const addressGet = "/api/releasenote";
+const addressGetSingle = "/api/releasenote/5";
+const addressGetSingleFail = "/api/releasenote/23123";
+const addressPut = "/api/releasenote/100";
+const addressPutFail = "api/releasenote/123123125";
+const addressDelete = "/api/releasenote/5";
 const addressDeleteFail = "/api/releasenote/897324";
 
+var noteToDelete;
+
 const init = () => {
   accessToken = TokenHandler.getAccessToken();
 };
 
+describe("Release notes GET", () => {
+  before(() => init());
+  // Should return a 401 unauth
+  it("GET | Get all notes without auth", done => {
+    chai
+      .request(process.env.APP_URL)
+      .get(addressGet)
+      .end(err => {
+        err.should.have.status(401);
+        done();
+      });
+  });
+
+  // should return 200 OK and a array of release notes
+  it("GET | Returns all the release note dummy data", done => {
+    chai
+      .request(process.env.APP_URL)
+      .get(addressGet)
+      .set("Content-Type", "application/json")
+      .set("Authorization", "Bearer " + accessToken)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+        res.should.have.status(200);
+        done();
+      });
+  });
+
+  // should return a 401 unauth
+  it("GET | Tries to get a single release without auth", done => {
+    chai
+      .request(process.env.APP_URL)
+      .get(addressGetSingle)
+      .end(err => {
+        err.should.have.status(401);
+        done();
+      });
+  });
+
+  // should return 200 OK and  a singe release note
+  it("GET | Successfully gets a single release note", done => {
+    chai
+      .request(process.env.APP_URL)
+      .get(addressGetSingle)
+      .set("Content-Type", "application/json")
+      .set("Authorization", "Bearer " + accessToken)
+      .end((err, res) => {
+        if (err) {
+          done(err.response);
+        }
+        res.should.have.status(200);
+        expect(res.body.id).to.equal(5);
+        noteToDelete = res.body;
```

Comment:
Samme som tidligere, legg inn sjekk for alle feltan.

Reply:
Ikke så lurt å sjekk mot eksakte verdia, gjør heller sånn som vi har gjort tidligere.
![image](https://user-images.githubusercontent.com/33400600/76299095-c7939b00-62ba-11ea-81e5-5a6cf6bd5aa5.png)

Her sjekk vi bære om de forskjellige feltan eksister og eller har en verdi.

Reply:
kor eksister den koden dar

Reply:
testReleases.js linja 221 -> 225